addons/mail/static/src/models/thread/thread.js

```diff
@@ -1192,6 +1262,26 @@ function factory(dependencies) {
             return [['replace', this.messages.filter(message => message.isNeedaction)]];
         }
 
+        /**
+         * @returns {integer|undefined}
+         * @private
+         */
+        _computeMessageBeforeNewMessageSeparatorId() {
+            if (this.model !== 'mail.channel') {
+                return clear();
+            }
+            if (this.localMessageUnreadCounter === 0) {
+                return clear();
+            }
+            if (!this.lastSeenByCurrentPartnerMessageId) {
+                return this.lastTransientMessage ? this.lastTransientMessage.id : 0;
+            }
+            if (!this.lastTransientMessage) {
+                return this.lastSeenByCurrentPartnerMessageId;
+            }
+            return Math.max(this.lastTransientMessage.id, this.lastSeenByCurrentPartnerMessageId);
```

Comment:
same remark then here about "last transient", it must be last transient before "lastSeenByCurrentPartnerMessageId"